plugins/temporal/temporal.go

```diff
@@ -17,13 +19,20 @@ type Config struct {
 
 type Temporal interface {
 	GetClient() (client.Client, error)
+	GetConfig() Config
 	CreateWorker(taskQueue string, options worker.Options) (worker.Worker, error)
 }
 
 type Provider struct {
+	endure.Named
+	// Rpc implementation
+	Rpc
```

Comment:
where is the interface?

Reply:
yeah, we do not need here a Named interface and RPC struct integration. The Provider struct implements roadrunner.rpc.Plugin interface.